src/swagger.json

```diff
@@ -0,0 +1,115 @@
+{
```

Comment:
It is necessary to implement Swagger Documentation by using Swagger JS Docs for generating API documentation (https://www.npmjs.com/package/swagger-jsdoc)